src/Template/Element/wall/message.ctp

```diff
@@ -0,0 +1,162 @@
+<?php
+use App\Lib\LanguagesLib;
+use App\Model\CurrentUser;
+
+$username = isset($message->user) ? $message->user->username : CurrentUser::get('username');
+$avatar = isset($message->user) ? $message->user->image : CurrentUser::get('image');
+$createdDate = $message->date;
+$modifiedDate = $message->modified;
+$messageId = $message->id;
+$authorId = $message->owner;
+$messageText = $message->content;
+$messageHidden = $message->hidden;
+$sentence = null;
+$sentenceOwnerLink = null;
+
+$editUrl = $this->Url->build(array(
+    'controller' => 'sentence_comments',
+    'action' => 'edit',
+    $messageId
+));
+
+if (empty($modifiedDate) || $createdDate == $modifiedDate) {
+    $dateLabel = $this->Date->ago($createdDate);
+    $fullDateLabel = $createdDate;
+} else {
+    $dateLabel = format(
+        __('{createdDate}, edited {modifiedDate}'),
+        array(
+            'createdDate' => $this->Date->ago($createdDate),
+            'modifiedDate' => $this->Date->ago($modifiedDate)
+        )
+    );
+    $fullDateLabel = format(
+        __('{createdDate}, edited {modifiedDate}'),
+        array(
+            'createdDate' => $createdDate,
+            'modifiedDate' => $modifiedDate
+        )
+    );
+}
+$canViewContent = CurrentUser::isAdmin() || CurrentUser::get('id') == $authorId;
+$userProfileUrl = $this->Url->build(array(
+    'controller' => 'user',
+    'action' => 'profile',
+    $username
+));
+
+if (isset($message['Permissions'])) {
+    $menu = $this->Wall->getMenuFromPermissions(
+        $message,
+        $message['Permissions']
+    );
+} else {
+    $menu = [];
+}
+$menu[] = [
+    'text' => 'permalink',
+    'icon' => 'link',
+    'url' => [
+        'controller' => 'wall',
+        'action' => 'show_message',
+        $messageId,
+        '#' => "message_" .$messageId
+    ]
+];
+
+$children = $message->children;
+$cssClass = isset($isRoot) ? 'wall-thread' : 'reply';
+$canReply = false;
+?>
+
+<md-card class="comment <?= $cssClass ?> <?= $messageHidden ? 'inappropriate' : '' ?>">
```

Comment:
Adding an id attribute here would allow linking to a specific message which was possible with the old design (see also the comments on the wall by [AlanF_US](https://tatoeba.org/wall/show_message/33384#message_33384) and [CK](https://tatoeba.org/wall/show_message/33385#message_33385)):

    <md-card id="message_<?= $messageId ?>" class=...

Reply:
I was about to do that :)